src/sdl/window.cpp

```diff
@@ -157,13 +166,41 @@ uint32_t window::get_flags()
 void window::set_minimum_size(int min_w, int min_h)
 {
 	SDL_SetWindowMinimumSize(window_, min_w, min_h);
+	// Can this change the size of the window?
```

Comment:
Is that a TODO before merging the PR? Just checking

Reply:
No, it will work fine in any case. I just wasn't sure if it was necessary to update the size variables when this was called. So i updated them anyway. If it doesn't actually change the window size the line i added can be removed. Not a huge issue either way.